docs/network/delayed-inbox.md

```diff
@@ -0,0 +1,106 @@
+
+## The delayed inbox
+
+The delayed inbox is responsible for two aspects:
+1. Allowing to bridge assets from the Tezos Layer1 to the Etherlink Smart Rollup.
+1. Force inclusion of transactions if the sequencer is censoring your transactions.
+
+### What is concrectly a delayed inbox
+
+In order to benefit from the low-latency and preconfirmation of the Etherlink network,
+transactions are generally forwarded to the sequencer. An alternative is to transmit the
+transactions via the delayed inbox directly on L1. These transactions are delayed in the
+sense that the workflow does the opposite of a regular transaction cycle in the sequencer.
+
+If you are using a sequencer, the general workflow is:
+1. The transaction is sent to the sequencer.
+1. Preconfirmation is given in less than 1 second.
+1. The transaction is part of a blueprint and published to L1.
+1. The transaction is validated by rollup nodes, 20s to 60sec later.
+
+If you use the delayed inbox, the general workflow is rather:
+1. The transaction is sent directly to the L1.
+1. The transaction is picked by rollup node and added to a specific storage "delayed-inbox".
+1. The sequencer monitors this this storage to discover new delayed transactions.
+1. The sequencer includes the potential delayed transaction using its hash.
+1. When the blueprint including a delayed transaction is published and validated by the
+rollup nodes, the transaction is removed from the delayed inbox storage.
+
+### Briding assets
+
+As described in the bridge section (todo link), Etherlink handles tickets forwarded from
+the Tezos L1. Therefore, it's not possible to simply ask the sequencer to bridge your assets
+on Etherlink. One kind of delayed inbox transaction is a `Deposit` request, when a deposit
+is included on the L1, they are added to the delayed inbox by the rollup nodes. Then the sequencer
+picks the deposits and include them in the block.
+
+### Overcoming censorship
+
+If the sequencer refuses to include your transaction for some reasons, the transaction can
+be submitted via the delayed inbox.
+
+The transaction must be transmitted via the "delayed bridge", a Tezos smart contract:
+1. Deployement on mainnet: [KT1AZeXH8qUdLMfwN2g7iwiYYSZYG4RrwhCj](https://better-call.dev/mainnet/KT1AZeXH8qUdLMfwN2g7iwiYYSZYG4RrwhCj)
+1. Deployement on ghostnet: [TODO](TODO)
+
+It's a special contract that takes two parameters:
+1. The Smart Rollup address of Etherlink.
+1. The rlp encoded transaction as bytes.
+
+When an user calls this contract it essentialy does two things:
+1. Charge the user 1XTZ to prevent spam on the network. The value 1XTZ is hardcoded and subject
+to evolution in the future.
+2. Forwards the transaction to the Etherlink Smart Rollup via the inbox.
+
+The contract is whitelisted in the storage of the Etherlink Smart Rollup, and everytime
+a transaction comes from this contract, it is added to the delayed inbox. Which can be picked
+up by the sequencer later.
```

Comment:
```suggestion
If the sequencer refuses to include a transaction for some reasons, the user can submit the transaction via the delayed inbox.
To put a transaction in the delayed inbox, the user sends the transaction to the "delayed bridge" contract on Tezos layer 1:

- Deployment on mainnet: [KT1AZeXH8qUdLMfwN2g7iwiYYSZYG4RrwhCj](https://better-call.dev/mainnet/KT1AZeXH8qUdLMfwN2g7iwiYYSZYG4RrwhCj)
- Deployment on ghostnet: [TODO](TODO)

This contract accepts two parameters:
- The address of the Etherlink Smart Rollup
- The RLP-encoded transaction in bytes

The user must include 1XTZ with the transaction as an anti-spam measure.
This value is hard-coded but could change in the future.

Then the contract forwards the transaction to the Etherlink Smart Rollup.
The delayed bridge contract is allowlisted in the storage of the Etherlink Smart Rollup, so when a transaction comes from this contract, Etherlink automatically adds it to the delayed inbox for the sequencer to process.
```

Reply:
^ How does the contract forward the transaction to the Smart Rollup? From the description above, I assumed that this contract would put the transaction directly into the delayed inbox. Is it a regular message in the regular Smart Rollup inbox, not the delayed inbox?